app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -0,0 +1,73 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <div class="field">
+    <label class="required">{{t 'Access Code'}}</label>
+    {{input type='text' name='access_code' value=access_code}}
+  </div>
+  <div class="field">
+    <label class="required">{{t 'Number of Access Tickets'}}</label>
+    {{input type='number' name='number_of_access_tickets' placeholder='0' min=0}}
+  </div>
+  <div class="grouped inline fields">
+    <label class="required">{{t 'Status'}}</label>
+    <div class="field">
+      {{ui-radio current='active' name='status' label='Active' value='active'}}
+    </div>
+    <div class="field">
+      {{ui-radio name='status' label='Inactive' value='inactive'}}
+    </div>
+  </div>
+  <div class="inline field">
+    <label class="required">{{t 'Select Tickets applied to the access code'}}</label>
+    <div class="ui hidden divider"></div>
+    {{ui-checkbox label='Select all Ticket types' name='all_ticket_types' checked=checked onChange=(action (mut checked))}}
+  </div>
+  <div>
+    <div class="inline field">
+      {{ui-checkbox class='toggle' label=(t 'Show more options') checked=showMoreOptions onChange=(action (mut showMoreOptions))}}
+    </div>
+    {{#if showMoreOptions}}
+      <div class="ui tiny header">{{t 'Set the min and max quantity allowed to purchase per order(Optional)'}}</div>
+      <div class="{{if device.isMobile '' 'fields'}}">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <div class="ui labeled input">
+            <div class="ui label">
+              {{t 'Min'}}
+            </div>
+            {{input type='number' name='min' min=0}}
+          </div>
+        </div>
+        {{#if device.isMobile}}
+          <div class="ui hidden divider"></div>
+        {{/if}}
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <div class="ui labeled input">
+            <div class="ui label">
+              {{t 'Max'}}
+            </div>
+            {{input type='number' name='max' min=0}}
+          </div>
+        </div>
+      </div>
+      <div class="ui hidden divider"></div>
+      <div class="ui tiny header">{{t 'Validity Period(Optional)'}}</div>
+      <div class="fields">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Valid from'}}</label>
+          {{widgets/forms/date-picker}}
+        </div>
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Expires on'}}</label>
+          {{widgets/forms/date-picker}}
```

Comment:
End date keep it 20 days from start date for now. Later we'll change this depending on event date